### PR TITLE
Fix /_admin/cluster/rebalance DocuBlocks

### PR DESCRIPTION
**Pull request by KVS85**
### Scope & Purpose

Fix /_admin/cluster/rebalance DocuBlocks.

- [x] :hankey: Bugfix

### Checklist

- [x] Backports
  - [x] Backport for 3.10: https://github.com/arangodb/arangodb/pull/16665

#### Related Information

- [x] Jenkins: https://jenkins.arangodb.biz/view/Documentation/job/arangodb-ANY-examples/2244/

### CODE REVIEW

File: Documentation/DocuBlocks/Rest/Administration/post_admin_cluster_rebalance.md [hunk 1]
```diff
@@ -4,25 +4,29 @@
 
 @RESTHEADER{POST /_admin/cluster/rebalance, Compute a set of move shard operations to improve balance.}
```
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Administration/post_admin_cluster_rebalance.md:5]: The summary seems too long and not end-user focused. A more detailed explanation should go into the `@RESTDESCRIPTION`, which every DocuBlock should have.

The wording leaves it open whether the endpoint actually executes a rebalancing or if it only analysis the situation and proposes how to move shards.

File: Documentation/DocuBlocks/Rest/Administration/post_admin_cluster_rebalance.md [hunk 2]
```diff
@@ -4,25 +4,29 @@
...
+@RESTBODYPARAM{piFactor,number,optional,}
 (Default `256e6`)
-@RESTBODYPARAM{databasesExcluded, array, optional,}
-List of database names that are excluded from the analysis.
 
-@RESTRETURNCODES
+@RESTBODYPARAM{databasesExcluded,array,optional,string}
+List of database names that are excluded from the analysis. (Default [])
+
+@RESTDESCRIPTION
```
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Administration/post_admin_cluster_rebalance.md:28]: The description should not be identical to the summary / brief.

Any reason as to why this was moved down, after the `@RESTBODYPARAM`s?

File: Documentation/DocuBlocks/Rest/Administration/post_admin_cluster_rebalance_execute.md [hunk 3]
```diff
@@ -8,9 +8,9 @@
 Execute the given set of move shard operations. Such a set can be calculated using
 other rebalance apis.
```
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Administration/post_admin_cluster_rebalance_execute.md:9]: How many are there? Should be more concrete.
```suggestion
other rebalance APIs.
```

File: Documentation/DocuBlocks/Rest/Administration/put_admin_cluster_rebalance.md [hunk 4]
```diff
@@ -2,27 +2,31 @@
...
-@RESTBODYPARAM{piFactor, number, optional,}
+
+@RESTBODYPARAM{piFactor,number,optional,}
 (Default `256e6`)
-@RESTBODYPARAM{databasesExcluded, array, optional,}
-List of database names that are excluded from the analysis.
 
-@RESTRETURNCODES
+@RESTBODYPARAM{databasesExcluded,array,optional,string}
+List of database names that are excluded from the analysis. (Default [])
```
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Administration/put_admin_cluster_rebalance.md:26]:
```suggestion
List of database names that are excluded from the analysis. (Default: `[]`)
```

File: Documentation/DocuBlocks/Rest/Administration/put_admin_cluster_rebalance.md [hunk 5]
```diff
@@ -2,27 +2,31 @@
 @startDocuBlock put_admin_cluster_rebalance
 @brief Compute a set of move shard operations to improve balance and immediately execute these.
 
-@RESTHEADER{POST /_admin/cluster/rebalance, Compute a set of move shard operations to improve balance and execute.}
+@RESTHEADER{PUT /_admin/cluster/rebalance, Compute a set of move shard operations to improve balance and execute.}
```
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Administration/put_admin_cluster_rebalance.md:5]: Summary seems too long

**Review by Simran-B — CHANGES_REQUESTED** [review]
Server replies (`@RESTRETURNCODES`) should be written in present tense.

See the last paragraph of the `@RESTBODYPARAM` about shared structs:
https://github.com/arangodb/docs#restbodyparam
There is already a `1_structs.md` in the Administration folder:
https://github.com/arangodb/arangodb/blob/devel/Documentation/DocuBlocks/Rest/Administration/1_structs.md

**Comment by KVS85** [comment]
@Simran-B we need to restore examples generation now. Please, propose your fixes to these DocuBlocks for devel and 3.10 in separate PRs.

**Review by maierlars — APPROVED** [review]

**Pull request merged**